lib/plugins/entities.js

```diff
@@ -212,6 +213,14 @@ function inject (bot) {
     entity.yaw = conv.fromNotchianYawByte(packet.yaw)
     entity.pitch = conv.fromNotchianPitchByte(packet.pitch)
     entity.objectData = packet.objectData
+
+    if (entity.type === 'player') {
```

Comment:
that looks pretty suspicious, not sure if it's right

Reply:
I suspect it is correct based on existing code using the opposite condition. https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/entities.js#L42

Not sure if what its being used for is correct tho.

Reply:
named_entity_spawn was removed, so players spawn with the generic spawn_entity packet. However there are alot of unhandled fields that are not currently accounted for - https://github.com/PrismarineJS/mineflayer/blob/a0a577763d08f44494669793f8f4a2f8b2d58e0f/lib/plugins/entities.js#L132